vgen/src/main/resources/io/gapi/vgen/go/main.snip

```diff
@@ -199,38 +211,23 @@
 
                     {@methodComment(method, methodName)}
                     func (client *Client) {@methodName}(ctx context.Context, req {@inTypeName}, opts ...gax.CallOption) (*{@iteratorTypeName}, error) {
-                    	callOpts := append({@retryParamsName}RetryParams(), client.CallOptions["{@methodName}"]...)
+                    	callOpts := append({@retryCodesMethodName}RetryCodes(), {@retryParamsMethodName}RetryParams(), client.CallOptions["{@methodName}"]...)
                     	callOpts = append(callOpts, opts...)
-                    	@if {@method.getResponseStreaming}
-                    	result, err := gax.Invoke(ctx, req, client.{@lowerMethodName}, callOpts...)
-                    	if err != nil {
-                    		return nil, err
-                    	}
-                    	receiver := result.({@context.getStreamReceiver(service, method)})
-                    	return &{@iteratorTypeName}{
-                    		apiCall: func() ({@iteratorPageTypeName}, error) {
-                    			resp, err := receiver.Recv()
-                    			if err != nil {
-                    				return {@iteratorPageTypeName}{}, err
-                    			}
-                    			return {@iteratorPageTypeName}{
-                    				Items: resp.{@resourceFieldName},
-                    				NextPageToken: resp.{@responseTokenFieldName},
-                    			}, nil
-                    		},
-                    	}, nil
-                    	@else
                     	atLastPage := false
                     	return &{@iteratorTypeName}{
                     		apiCall: func() ({@iteratorPageTypeName}, error) {
                     			if atLastPage {
                     				return {@iteratorPageTypeName}{}, io.EOF
                     			}
-                    			result, err := gax.Invoke(ctx, req, client.{@lowerMethodName}, callOpts...)
+                    			var resp {@outTypeName}
+                    			err := gax.Invoke(ctx, func (childCtx context.Context) error {
```

Comment:
We can just call the parameter "ctx" instead of "childCtx". That's more idiomatic Go..